Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/activities/EmulationActivity.java

```diff
@@ -289,6 +298,12 @@ protected void onCreate(Bundle savedInstanceState)
     if (mEmulationFragment == null)
     {
       mEmulationFragment = EmulationFragment.newInstance(mPaths, mRiivolution);
+      if (mIsSystemMenu)
+      {
+        Bundle bundle = new Bundle();
+        bundle.putBoolean("systemMenu", true);
+        mEmulationFragment.setArguments(bundle);
```

Comment:
It is preferable if this kind of logic is put in a static function inside `EmulationFragment`. That way the fact that the relevant key is called "systemMenu" never has to be known outside of `EmulationFragment`.